src/TaylorSeries.jl

```diff
@@ -49,7 +49,8 @@ export getcoeff, derivative, integrate,
     get_order, get_numvars,
     set_variables, get_variables,
     get_variable_names, get_variable_symbols,
-    ∇, jacobian, jacobian!, hessian, hessian!,
+    # jacobian, hessian,
+    ∇, jacobian!, hessian!,
```

Comment:
Why are these still exported?

Reply:
They are in-place versions of `jacobian` and `hessian`; they are still exported since I think there is no name-conflict risk.